plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/FunnyCommandsConfiguration.java

```diff
@@ -146,6 +146,7 @@ public static FunnyCommands createFunnyCommands(FunnyGuilds plugin) {
                 .completer(new GuildsCompleter(guildManager))
                 .completer(new AlliesCompleter(userManager))
                 .completer(new GuildInvitationsCompleter(userManager, plugin.getGuildInvitationList()))
+                .completer(new InviteCommand.InviteCommandCompleter(config, userManager))
```

Comment:
Skoro to jest w środku i tak, to w sumie czy jest potrzeba prefiksować? Kwestia stylu, ale zazwyczaj w takiej sytuacji staram się robić rzeczy typu `InviteCommand.Completer`, inaczej wychodzą potworki, a nie linijki.